cf-testd/cf-testd.c

```diff
@@ -239,17 +224,18 @@ static bool CFTestD_ProtocolError(
     strcpy(sendbuffer, "BAD: Request denied");
     SendTransaction(conn->conn_info, sendbuffer, 0, CF_DONE);
     Log(LOG_LEVEL_INFO,
-        "Closing connection due to illegal request: %s", recvbuffer);
+        "Closing connection due to illegal request: %s",
+        recvbuffer);
     return false;
 }
 
 static bool CFTestD_BusyLoop(ServerConnectionState *conn)
 {
-    char recvbuffer[CF_BUFSIZE + CF_BUFEXT] = "";
+    char recvbuffer[CF_BUFSIZE + CF_BUFEXT]        = "";
     char sendbuffer[CF_BUFSIZE - CF_INBAND_OFFSET] = "";
 
-    const int received = ReceiveTransaction(conn->conn_info,
-                                            recvbuffer, NULL);
+    const int received =
+        ReceiveTransaction(conn->conn_info, recvbuffer, NULL);
```

Comment:
Again, I liked the original better.

Reply:
In this specific case, I somewhat agree, but I think they're very close. I'm not going to force others to use clang-format, manual formatting should still be an option. However, I really like that clang-format, with the current config, follows a few very simple and predictable rules.

Reply:
Yeah, just please limit its effect to the new and changed code.